.github/workflows/deploy-staging.yml

```diff
@@ -0,0 +1,96 @@
+name: "Deploy - Staging"
+on: 
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Branch name"
+        type: string
+        required: true
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+defaults:
+  run:
+    shell: bash
+
+jobs:
+
+  release-docker:
+    name: "Release Docker image"
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Print inputs
+        run: echo "Building docker image from ${{ github.event.inputs.branch }} branch..."
+
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.inputs.branch }}
+      
+      - name: Install DigitalOcean CLI
+        uses: digitalocean/action-doctl@v2
+        with:
+          token: ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}
+      
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        id: buildx
+        with:
+          version: latest
+      
+      - name: Login to DigitalOcean Container Registry
+        run: doctl registry login --expiry-seconds 600
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Configure Docker image metadata
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            name=ghcr.io/${{ github.repository}}
+            name=registry.digitalocean.com/cheqd/${{ github.repository }}
+          tags: |
+            type=raw,value=staging-latest
+          labels: |
+            org.opencontainers.image.title="cheqd DID Resolver"
+            org.opencontainers.image.description="DID Resolver for the did:cheqd method"
+            org.opencontainers.image.source="https://github.com/cheqd/did-resolver"
+            org.opencontainers.image.vendor="Cheqd Foundation Limited"
+            org.opencontainers.image.created={{date 'dddd, MMMM Do YYYY, h:mm:ss a'}}
+            org.opencontainers.image.documentation="https://docs.cheqd.io/identity"
+
+      - name: Build and push image
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          file: docker/Dockerfile
+          platforms: linux/amd64
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          
+  deploy-staging-resolver:
+    name: "Deploy DID-resolver to the staging environment"
+    needs: release-docker
+    runs-on: ubuntu-latest
+    environment:
+      name: staging
+      url: https://did-resolver-dev-39yxa.ondigitalocean.app
```

Comment:
Just make this a simpler, custom URL. Please set it to resolver-staging.cheqd.net rather than this random DO one or the .dev one.

Reply:
Agree.

Reply:
Then change it?

Reply:
I did it here [https://github.com/cheqd/did-resolver/commit/504886411a3d6a358ab0302f666d68d9bcabe7af](url), PR is not updated yet.